src/server/mod.rs

```diff
@@ -137,7 +137,7 @@ impl Server {
 
             vec![build_app(state), build_dockerflow(dfstate)]
         })
-        .bind(format!("127.0.0.1:{}", settings.port))
+        .bind(format!("{}:{}", settings.host, settings.port))
```

Comment:
FYI `.bind((settings.host.as_ref(), settings.port))` also works, there's a ToSocketAddr's impl for that kind of 2 item tuple

Reply:
Yeah, wasn't sure if "127.0.0.1" was something explicitly set, otherwise would have done something like that.